packages/nimble-components/src/anchor-tabs/index.ts

```diff
@@ -0,0 +1,297 @@
+// Based on:
+// https://github.com/microsoft/fast/blob/6bce27d0b2d654650b8751bf055f5e3b5a4f9250/packages/web-components/fast-foundation/src/tabs/tabs.ts
+// The code is heavily modified such that a diff against the original is useless.
+// Primary differences are:
+// - Reimplemented active tab indicator and removed attribute to disable it
+// - Separated tab focus and tab selection, i.e. can focus a different tab than the selected one
+// - Removed everything related to tab panels
+// - Removed support for vertical tab orientation
+// - Removed change event
+// - Conforms to our linter rules
+import { attr, observable } from '@microsoft/fast-element';
+import {
+    keyArrowLeft,
+    keyArrowRight,
+    keyEnd,
+    keyEnter,
+    keyHome,
+    keySpace,
+    uniqueId
+} from '@microsoft/fast-web-utilities';
+import {
+    DesignSystem,
+    StartEnd,
+    applyMixins,
+    StartEndOptions,
+    FoundationElementDefinition,
+    FoundationElement
+} from '@microsoft/fast-foundation';
+import { styles } from './styles';
+import { template } from './template';
+import type { AnchorTab } from '../anchor-tab';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-anchor-tabs': AnchorTabs;
+    }
+}
+
+export type TabsOptions = FoundationElementDefinition & StartEndOptions;
+
+/**
+ * A nimble-styled set of anchor tabs
+ */
+export class AnchorTabs extends FoundationElement {
+    /**
+     * The id of the active tab
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: activeid
+     */
+    @attr
+    public activeid = '';
```

Comment:
This should be optional right? You don't have to set an activeId

```suggestion
    public activeid?;
```

The problem with setting a default of empty string is that reflects to the DOM .

Reply:
Changed